src/api/java/com/minecolonies/api/client/render/modeltype/BipedModelType.java

```diff
@@ -1,78 +1,72 @@
 package com.minecolonies.api.client.render.modeltype;
 
+import com.minecolonies.api.entity.citizen.AbstractEntityCitizen;
+import net.minecraft.util.ResourceLocation;
+
 /**
  * Enum with possible citizens.
  */
-public enum BipedModelType implements ISimpleModelType
+public class BipedModelType implements ISimpleModelType
```

Comment:
This should stay an enum.

You should never need this in the API anyway.... everwhere where it is in use in the API or internally it should use the ISimpleModelType interface.